lib/ui/chat/widgets/message_widget.dart

```diff
@@ -106,229 +104,245 @@ class MessageWidget extends StatelessWidget {
   }
 
   Widget _buildMessageContent(BuildContext context) {
-    return LayoutBuilder(
-      builder: (context, constraints) {
-        return ConstrainedBox(
-          constraints: BoxConstraints(
-            maxWidth: constraints.maxWidth,
+    final screenWidth = MediaQuery.of(context).size.width;
```

Comment:
Not a blocker but FYI, it's preferred to use MediaQuery.sizeOf(context).width as it's [more efficient](https://api.flutter.dev/flutter/widgets/MediaQuery-class.html) (since we don't need to recalculate on device rotation).